vuepress/docs/next/docs/getting-started/ent-profile.md

```diff
@@ -1,3 +1,92 @@
 ---
 sidebarDepth: 1
 ---
+
+
+# Profile Management
+
+The [Entando CLI](entando-cli.md) can define a collection of configuration variables to full describe an Entando instance. One or more profiles can be created to store configuration settings for the instance. The `ent profile` command is available to manage and switch between the configurations of different Entando instances. Refer to `ent profile first-use-readme` for additional details. A profile with this information is automatically generated for a quickstart application.
```

Comment:
the first revision is not correct. the ent cli does not do this automatically, and unless you're running a quickstart app a profile is not created for you, so there is no profile that exists. also, the profile defines what entando instance to use, it is not used by the instance.

the second revision loses appreciable clarification. and profiles define an instance, they don't exert any governance like perform switching.

Reply:
Makes sense.  I was mistaken about the way profile is used. It is confusing with the config command. I think it was the term collection and the repetition of the word configuration that felt in need of simplifying. I think switching between profiles is a common thing and this section felt a bit complex. 

The term collection seems heavy handed because it is just mapping like 5 variables. Same with 'fully describe an Entando instance', the profile doesn't really describe the instance, just some of the parameters.

The `ent profile` command is available to manage and switch between the configurations of different Entando instances. >> Is it ent pro or ent profile? I think this sentence could be simplified to say you are switching between profiles, more direct and simple

Reply:
i think that "can define a collection of configuration variables" is the most precise phrasing. the cli can be used to resolve the ent env completely. the configuration variables necessary for / included in the profile are a small subset of that collection.

switching between profiles is very common, but entando switches between "profile and configuration" within commands, so it's important to describe the architecture/process. an instance can map to a profile. a profile has a configuration defined by a set of variables. the configuration, or configuration of the profile, is just one aspect of the profile. the profile itself includes much more that isn't defined by the configuration. further simplifications of what is written are misleading and not 100% accurate.

`ent pro` is shorthand for `ent profile`